Add language server section (#928)
ballerina-platform/ballerina-release: release-notes/swan-lake-beta3-release-note.md

#### New Features

##### Language Server

> - Add code action to pull modules from the Ballerina Central

Reviewer: ```suggestion
- Added a code action to pull modules from the Ballerina Central
```

> - Add Completion extension API for toml configuration files

Reviewer: ```suggestion
- Added the completion extension API for TOML configuration files
```

> - Add completion support for `Ballerina.toml`

Reviewer: ```suggestion
- Added completion support for the `Ballerina.toml` file
```

> - Add offline build configuration option

Reviewer: ```suggestion
- Added the offline build configuration option
```

> Clients can set the `ls.compilation.online` system property `true` or `false`, in order to run the language server's compilations online or offline. By default, the compilations are running offline. 

Reviewer: ```suggestion
Clients can set the `ls.compilation.online` system property to`true` or `false` in order to run the language server's compilations online or offline. By default, the compilations are running offline. 
```
